code/game/objects/items/shields.dm

```diff
@@ -44,12 +77,20 @@
 	item_state = "roman_shield"
 	lefthand_file = 'icons/mob/inhands/equipment/shields_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/shields_righthand.dmi'
+	transparent = FALSE
+	materials = list(MAT_METAL=8500)
+	max_integrity = 65

```

Comment:
I feel like this should be a lot lower, given it's just a costume item

Reply:
these are the ones used by necromantic stone thralls (along with a force 40 claymore and other gear), so wouldn't call them costume items and their `/fake` subtype has a 0% block chance anyway
i gave them 65 `max_integrity` instead of the default 75 becasue they will still block beam projectiles which makes them better than regular riot shields

Reply:
overwrite `max_integrity` for the `/fake`?